src/ai/backend/manager/idle.py

```diff
@@ -302,22 +321,61 @@ async def check_session(self, session: Row, dbconn: SAConnection) -> bool:
 
 class UtilizationIdleChecker(BaseIdleChecker):
     """
-    Checks the idleness of a session by the current utilization of all
-    compute devices and agents assigned to it.
+    Checks the idleness of a session by the average utilization of compute devices.
     """
 
     name: ClassVar[str] = "utilization"
-    default_cpu_util_threshold: ClassVar[float] = 30.0
-    default_accelerator_util_threshold: ClassVar[float] = 10.0
-    default_initial_grace_period: ClassVar[float] = 300.0  # allow first 5 minutes to be idle
+    check_interval: float = 20.0
+
+    _config_iv = t.Dict(
+        {
+            t.Key("time-window", default="10m"): tx.TimeDuration(),
+            t.Key("initial-grace-period", default="5m"): tx.TimeDuration(),
```

Comment:
You introduced `initial-grace-period` configs in Etcd, but I cannot find any code to reference this config.